src/geoops/macros.jl

```diff
@@ -66,19 +66,23 @@ _getcolexpr(expr) = :(getproperty(geotable, $(_nameexpr(expr))))
 function _colnames(expr)
   colnames = []
   if expr isa Expr
```

Comment:
@eliascarv isn't the function `_colnames` always called with expressions?

Reply:
No, sometimes, the RHS expression can be a constant value:
![image](https://github.com/user-attachments/assets/3d88d245-25b8-4643-aac9-831bb3cebf23)